preview/use-cases.md

```diff
@@ -0,0 +1,98 @@
+
+```csharp
+using Microsoft.Extensions.Hosting;
+
+public class Program
+{
+    public static void Main(string[] args)
+    {
+        CreateHostBuilder(args).Build().Run();
+    }
+
+    public static IHostBuilder CreateHostBuilder(string[] args) =>
+        Host.CreateDefaultBuilder(args)
+            .ConfigureAppConfiguration((context, config) => 
+            {
+                config.AddJsonFile("appsettings.json")
+                      .AddJsonFile("appsettings.Development.json");
+            })
+            .ConfigureSecretStore((context, config, builder) =>
+            {
+#if DEBUG
+                builder.AddConfiguration(config);
+#endif
+                var keyVaultName = config["KeyVault_Name"];
+                builder.AddEnvironmentVariables()
+                       .AddAzureKeyVaultWithManagedServiceIdentity($"https://{keyVaultName}.vault.azure.net");
+            });
+    }
+}
+```
+
+For more information, see [our documentation site](https://security.arcus-azure.net/features/secret-store).
+
+Blogs:
+* [Introducing Arcus secret store - making secrets a first-class citizen](https://www.codit.eu/blog/introducing-secret-store-net-core/)
+* [Advanced Arcus secret store features](https://www.codit.eu/blog/secret-store-arcus-security-v1-4/)
+
+## Simplify telemetry tracking with contextual information
+Tracking telemetry with useful information is not an easy task. Giving each telemetry instance enough contextual information, correlated with other telemetry... There are many platforms and logging systems that are doing a part of the work but not one system that combines them all.
+
+[Arcus Observability](https://observability.arcus-azure.net/) is the answer. The library provides a custom [Azure Application Insights Serilog sink](https://observability.arcus-azure.net/Features/sinks/azure-application-insights) that combines the easy application logging from Serilog with the highly expanded contextual information of Azure Application Insights. The library extends Serilog in such a way that all Azure Application Insights features can be logged with the well-known `ILogger` abstraction.
+
+```csharp
+ILogger logger = ...
+
+// Contextual information
+var telemetryContext = new Dictionary<string, object>
+{
+    { "Customer", "Contoso"},
+};
+
+// Events
+logger.LogEvent("Order Created", telemetryContext);
+
+// Dependencies
+using var measurement = DurationMeasurement.Start();
+logger.AzureKeyVaultDependency(vaultUri: "https://my-secret-store.vault.azure.net", secretName: "ServiceBus-ConnectionString", isSuccessful: true, measurement, telemetryContext);
+
+// Requests
+using var measurement = DurationMeasurement.Start();
+logger.LogRequest(httpContext.Request, httpContext.Response, measurement, telemetryContext);
+
+// Metrics
+logger.LogMetric("Orders Received", 133, telemetryContext);
+```
+
+For more information, see [our documentation site](https://observability.arcus-azure.net/).
```

Comment:
Maybe also mention this is already implemented in our templates, for example [templates.arcus-azure.net/features/azurefunctions-http-template](https://templates.arcus-azure.net/features/azurefunctions-http-template).

Reply:
https://github.com/arcus-azure/arcus/pull/204/commits/1a05fc739cbc183aebc33fb5d58e27a2283a0709

Reply:
https://github.com/arcus-azure/arcus/pull/204/commits/1a05fc739cbc183aebc33fb5d58e27a2283a0709